src/Dflydev/FigCookies/Cookies.php

```diff
@@ -71,28 +72,26 @@ public function without(string $name) : Cookies
     /**
      * Render Cookies into a Request.
      */
-    public function renderIntoCookieHeader(RequestInterface $request) : RequestInterface
+    public function renderIntoCookieHeader(RequestInterface $request): RequestInterface
     {
         $cookieString = implode('; ', $this->cookies);
 
-        $request = $request->withHeader(static::COOKIE_HEADER, $cookieString);
+        $request = $request->withHeader(self::COOKIE_HEADER, $cookieString);
 
         return $request;
     }
 
     /**
      * Create Cookies from a Cookie header value string.
-     *
-     * @return static
      */
-    public static function fromCookieString(string $string) : self
+    public static function fromCookieString(string $string): self
     {
-        return new static(Cookie::listFromCookieString($string));
+        return new self(Cookie::listFromCookieString($string));
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.